server/routes/auth.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const router = express.Router();
+// middleware for future protected routes
+const protected = require("../middleware/protectRoute");
+const {
+  register,
+  login,
+  sendPasswordResetCode,
+  resetPassword,
+  signInWithGoogle,
+  changePassword
+} = require("../controllers/auth");
+
+router.post("/register", register);
+router.post("/login", login);
+router.post("/loginwithgoogle", signInWithGoogle);
```

Comment:
Please use hyphens (-) to improve the readability of URIs, login-with-google looks neater:)